cloudinit/config/schema.py

```diff
@@ -163,10 +164,9 @@ def is_schema_byte_string(checker, instance):
 
 
 def _add_deprecation_msg(description: Optional[str] = None):
-    msg = "DEPRECATED."
     if description:
-        msg += f" {description}"
-    return msg
+        return f"{DEPRECATED_PREFIX}{description}"
+    return DEPRECATED_PREFIX.replace(":", ".").strip()
```

Comment:
Where descriptiion is empty, we want to end with `DEPRECATED.` instead of `DEPRECATED:`.